### PR TITLE
Kerem/fetch service eslint

### PR DESCRIPTION
**Pull request by mechanical-turk**
enable eslint and fix lint issues

Authors: Kerem + Tak

### CODE REVIEW

File: packages/stream-metadata/src/environment.ts [hunk 1]
```diff
@@ -1,59 +1,33 @@
...
-				chainId: envConfig.river.chainId,
-				riverRegistry: envConfig.river.addresses.riverRegistry,
-			}
-		}
-	}
-
-	if (!allChainConfig[riverEnv].chainId || !allChainConfig[riverEnv].riverRegistry) {
-		throw new Error('chainId or riverRegistry undefined')
-	}
+export const config = makeConfig(process.env.RIVER_ENV, process.env.RIVER_CHAIN_RPC_URL)
```
Comment (texuf) [inline packages/stream-metadata/src/environment.ts:14]: is this PR properly merged? I thought these changes went in already.

File: packages/stream-metadata/src/evmRpcClient.ts [hunk 2]
```diff
@@ -1,44 +1,24 @@
...
-		case 31338:
-			riverChainUrl = riverChainLocalhost.rpcUrls.default.http[0]
-			return createPublicClient({
-				chain: riverChainLocalhost,
-				transport: http(riverChainUrl),
-			})
-		default:
-			console.error(`Unsupported chain ID: ${chainId}`)
-			throw new Error(`Unsupported chain ${chainId}`)
+function createRiverRegistry(config: Config) {
```
Comment (jterzis) [inline packages/stream-metadata/src/evmRpcClient.ts:8]: This function exists in web3 https://github.com/river-build/river/blob/main/packages/web3/src/RiverRegistryFactory.ts#L5

File: packages/stream-metadata/src/environment.ts (hunk 1 above)
Comment (tak-hntlabs) [reply packages/stream-metadata/src/environment.ts]: not yet - this is a stacked PR. Awaiting parent PRs to merge first.

Comment (tak-hntlabs) [reply packages/stream-metadata/src/environment.ts]: Review in this order:

1. https://github.com/river-build/river/pull/727 - make env  & config deployment-ready
2. https://github.com/river-build/river/pull/729 - pino logger
3. https://github.com/river-build/river/pull/726 - eslint and lint fixes

**Review by texuf — APPROVED** [review]
looks good once it's green

File: packages/stream-metadata/package.json [hunk 3]
```diff
@@ -7,9 +7,10 @@
 	"type": "module",
 	"scripts": {
 		"build": "node ./esbuild.config.mjs",
-		"dev": "yarn build && NODE_ENV=development node --experimental-detect-module ./dist/node_esbuild.cjs",
+		"lint": "eslint . --ext .ts",
+		"dev": "yarn build && node --experimental-detect-module ./dist/node_esbuild.cjs",
 		"prettier": "prettier --config ../prettier-config/config.js --ignore-path .prettierignore .",
-		"start": "NODE_ENV=production node --experimental-detect-module ./dist/node_esbuild.cjs"
+		"start": "node --experimental-detect-module ./dist/node_esbuild.cjs"
```
Comment (mechanical-turk) [inline packages/stream-metadata/package.json:13]: @tak-hntlabs can we remove --experimental-detect-module? node should work with .cjs modules by default

Comment (tak-hntlabs) [reply packages/stream-metadata/package.json]: fixed

**Pull request merged**